javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber < 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  while (currentNumber < 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+  while (count < times) {
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
```

Comment:
`repeatWord` should include `times` copies of `word`, not `times - 1` copies:

---

![image](https://user-images.githubusercontent.com/16143771/152254743-9ce58f60-8a3f-4b00-b7bd-32f2f4a5e4fe.png)

---

![image](https://user-images.githubusercontent.com/16143771/152254786-97428b1c-ccfa-42cb-ac58-2dc5975ea129.png)